### DISCUSSION TITLE
label里面可不可以使用html啊，这样可以扩展更多的功能，比如一人跟随的时候，还可以展示行车记录仪上的实时数据

### DISCUSSION THREAD

**Discussion by Wd13869167203** [zouyaoji/vue-cesium#356] [General]
![image](https://user-images.githubusercontent.com/31448471/219576579-92d402a4-f4c7-437d-8bf2-1ecbac8254ae.png)

**Comment by zouyaoji** [comment]
可以结合vc-overlay-html结合一起用。
